### DISCUSSION TITLE
Améliorer la gestion de mails

### DISCUSSION THREAD

**Discussion by hfroot** [betagouv/ma-cantine#1365] [Idées]
Les mails qu'on reçois maintenant :

- copie de la requête de rejoindre l'équipe de gestionnaires d'une cantine
- requêtes d'aide pour l'import massif
- message en attente de validation
- cantine publiée

Est-ce qu'on veut déplacer/supprimer une partie/tous vers trello ? @gauthierandre tu trouves que trello marche bien pour les contacts ?

On peut créer une colonne "Informatives" pour les cartes qui n'ont pas besoin d'action de notre part, qui sont simplement pour info. Pour les mails vers l'équipe de gestion, s'il n'y a aucun gestionnaire, on mettra la carte créée dans la colonne "Nouveau demandes".

**Comment by gauthierandre** [comment]
Bof. Pour le trello contact par exemple, je suis le seul à l'utiliser au final. Et j'avoue ne pas le regarder tous les jours.
Côté bizdev on a pas mal d'outils, dont des trellos, et il y a parfois des loupés en terme de visibilité / consultation. 
Pas très efficace.
Je comprends le sens de ta demande, et clairement je pense qu'on peut réfléchir à soulager la boîte contact de pas mal d'éléments, avec une seconde adresse ?

**Reply by hfroot** [reply]
Merci pour tes retours, je vais en penser plus :)

**Comment by hfroot** [comment]
Je pense quand même qu'on peut enlever les copies de "cantine publiée" et "requête de rejoindre l'équipe" - est-ce que ces mails donnent des infos utiles ?

**Reply by gauthierandre** [reply]
Yes, on peut enlever les copies de cantines publiées ! 
Par contre, les "requête de rejoindre l'équipe" c'est encore important du fait de la campagne avec le plan de relance car des gens ne suivent pas tout à fait le lien qu'on leur a envoyé (car mail différent). 
Par contre, je serai carrément ok pour qu'on se fasse un point équipe adresse mail ! Peut-être au séminaire ou avant.
